### ISSUE TITLE
High Delay In Giving Information

### ISSUE THREAD

**Issue by SinaAzad** [EDCD/EDDI#568]
## EDDI version in which issue found
Standalone Version 3.0.1-b2

## Steps to reproduce

Traveling to another system

## Expected
By the time you are in the middle of the jump, it should be giving you the information about the destination system.

## Observed
It takes so long for it to give the information that I've already jumped to the 2nd destination and getting ready to jump to the next one.

## Investigation
None at the moment but It was not like this on the older version I used a few months ago.

**Comment by SinaAzad** [comment]
I'm not sure about that, the cause could be related but I don't feel like Eddi loading slower than usual, It is just that the given information about the destination start is few jumps late. by the way, other things are as good as they used to be, like selling stuff or reading messages they are real-time with no obvious delays.

**Comment by SinaAzad** [comment]
I did not see any differences after doing that.

**Comment by Tkael** [comment]
In that case, I'm not sure what's going on there.  Are you using the default `FSD engaged` script? There's a pause (5 seconds by default) built into the beginning of the script that you can adjust up or down as needed:
```
    {_ Wait for the FSD countdown and initial jump}
    {Pause(5000)}
```
5000 is the time to wait in milliseconds before proceeding.

We do have a large number of bug fixes queued for the next release though and we are intending to do a next release soon. I'll be interested to know if you observe any change with that new release.

**Comment by DaftMav** [comment]
Just wanted to add I'm experiencing the same delayed response, seems to happen only with the system information after jumping. Started happening with 3.0.1-b2, using default FSD script.

Doesn't happen on every jump though, but even using the 'Test' button on the FSD engaged script is delayed when it does. Kind of freezes the program too as I can't switch to a different tab until it catches up and finally starts with the speech response.

**Comment by DaftMav** [comment]
Had another delayed response on jumping to another system, this time I got the log details. See attached file, hopefully this helps in narrowing it down.

[eddi.log.delayed.jump.txt](https://github.com/EDCD/EDDI/files/2020686/eddi.log.delayed.jump.txt)

**Comment by Tkael** [comment]
Thanks. The log helps solidify my belief that these are issues that will be addressed in the next release.

**Comment by Tkael** [comment]
Closing as resolved in 3.0.1-b4
